README.md

```diff
@@ -1,59 +1,162 @@
-# Quickstart in Couchbase with Java and Spring Boot
-#### Build a REST API with Couchbase's Java SDK 3 and Spring Boot
+# Quickstart in Couchbase with Spring Boot and Java
 
-> This repo is designed to teach you how to connect to a Couchbase cluster to create, read, update, and delete documents and how to write simple parametrized N1QL queries.
+#### REST API using Couchbase Capella in Java using Spring Boot
 
-[![Try it now!](https://da-demo-images.s3.amazonaws.com/runItNow_outline.png?couchbase-example=java-springboot-quickstart-repo&source=github)](https://gitpod.io/#https://github.com/couchbase-examples/java-springboot-quickstart)
+Often, the first step developers take after creating their database is to create a REST API that can perform Create, Read, Update, and Delete (CRUD) operations for that database. This repo is designed to teach you and give you a starter project (in Java using Spring Boot) to generate such a REST API. After you have installed the travel-sample bucket in your database, you can run this application which is a REST API with Swagger documentation so that you can learn:
+
+1. How to create, read, update, and delete documents using Key-Value[ operations](https://docs.couchbase.com/java-sdk/current/howtos/kv-operations.html) (KV operations). KV operations are unique to Couchbase and provide super fast (think microseconds) queries.
+2. How to write simple parametrized [N1QL queries](https://docs.couchbase.com/java-sdk/current/howtos/n1ql-queries-with-sdk.html) using the built-in travel-sample bucket.
+
+Full documentation for the tutorial can be found on the [Couchbase Developer Portal](https://developer.couchbase.com/tutorial-quickstart-spring-boot-java/).
+
+If you are looking for a quick start using [Micronaut](https://micronaut.io/)](https://micronaut.io/), you can find it in this [repo](https://github.com/couchbase-examples/java-quickstart-micronaut).
 
-Full documentation can be found on the [Couchbase Developer Portal](https://developer.couchbase.com/tutorial-quickstart-java-springboot/).
 ## Prerequisites
+
 To run this prebuilt project, you will need:
 
-- Couchbase Server 7 Installed (version 7.0.0-5247 or higher)
-- Java SDK v1.8 or higher installed 
-- Code Editor installed (IntelliJ IDEA, Eclipse, or Visual Studio Code)
-- Maven command line
+- [Couchbase Capella](https://www.couchbase.com/products/capella/) cluster with [travel-sample](https://docs.couchbase.com/java-sdk/current/ref/travel-app-data-model.html) bucket loaded.
+  - To run this tutorial using a self-managed Couchbase cluster, please refer to the [appendix](#running-self-managed-couchbase-cluster).
+- [Java](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html) 11 or higher installed
+  - Ensure that the Java version is compatible with the Couchbase SDK.
```

Comment:
Can you link to the SDK docs here: https://docs.couchbase.com/java-sdk/current/project-docs/compatibility.html#jdk-compat

Reply:
done